src/query/interpreter.cpp

```diff
@@ -2923,7 +2934,12 @@ void Interpreter::Abort() {
 
   expect_rollback_ = false;
   in_explicit_transaction_ = false;
+
+  memgraph::metrics::IncrementCounter(memgraph::metrics::RollbackedTransactions);
```

Comment:
I think Abort() function is called from multiple places and doesn't necessarily mean rollback. How about moving this DecrementCounter to the Rollback function?

Reply:
I can move it yes. Makes sense